PUSHDOWN.md

```diff
@@ -0,0 +1,321 @@
+
+To enable Query Pushdown from Java/Scala, you will first need to include the connector as a dependency. The instructions for the same are [here](https://github.com/GoogleCloudDataproc/spark-bigquery-connector#compiling-against-the-connector). Then, you can call `com.google.cloud.spark.bigquery.BigQueryConnectorUtils.enablePushdownSession(spark)` to enable Query Pushdown for your Spark session from your project code.
+
+If you want to disable Query Pushdown for the already enabled Spark session, you can call `com.google.cloud.spark.bigquery.BigQueryConnectorUtils.disablePushdownSession(spark)`
+
+### Validating/Debugging Query Pushdown
+
+If you experience any issues with Query Pushdown, you can use `df.explain()` on the DataFrame. If you pass the argument true you will get a lot more output that includes pre and post optimization phases.
+
+You can be sure that the entire query has been pushed down if you only see Spark(Version)BigQueryPushdownPlan as part of the physical plan output.
+
+**Note:** If you are running jobs on Dataproc using the connector, the job logs will also give information about any errors encountered during query pushdown
+
+## Supported Pushdown Operations
+
+The following Spark functions/operations are supported in Query Pushdown.
+
+### Aggregation Functions
+
+| Spark function name | Spark SQL function name                                                             |
+|---------------------|-------------------------------------------------------------------------------------|
+| Average             | [avg](https://spark.apache.org/docs/latest/api/sql/index.html#avg)                  |
+| Corr                | [corr](https://spark.apache.org/docs/latest/api/sql/index.html#corr)                |
+| CovPopulation       | [covar_pop](https://spark.apache.org/docs/latest/api/sql/index.html#covar_pop)      |
+| CovSample           | [covar_samp](https://spark.apache.org/docs/latest/api/sql/index.html#covar_samp)    |
+| Count               | [count](https://spark.apache.org/docs/latest/api/sql/index.html#count) 	            |
+| Max                 | [max](https://spark.apache.org/docs/latest/api/sql/index.html#max)                  |
+| Min                 | [min](https://spark.apache.org/docs/latest/api/sql/index.html#min)                  |
+| StddevPop           | [stddev_pop](https://spark.apache.org/docs/latest/api/sql/index.html#stddev_pop)    |
+| StddevSamp          | [stddev_samp](https://spark.apache.org/docs/latest/api/sql/index.html#stddev_samp)  |
+| Sum                 | [sum](https://spark.apache.org/docs/latest/api/sql/index.html#sum)                  |
+| VariancePop         | [var_pop](https://spark.apache.org/docs/latest/api/sql/index.html#var_pop)          |
+| VarianceSamp        | [var_samp](https://spark.apache.org/docs/latest/api/sql/index.html#var_samp)        |
+
+### Boolean Functions
+
+| Spark function name   | Spark SQL function name                                                           |
```

Comment:
Please change to the relevant operator (and - &&, contains - `LIKE '%foo%'` etc.)

Reply:
Changed `Spark function name`  to `Spark Logical Plan Construct`